src/node_env_var.cc

```diff
@@ -389,13 +390,32 @@ static void EnvEnumerator(const PropertyCallbackInfo<Array>& info) {
       env->env_vars()->Enumerate(env->isolate()));
 }
 
+static void EnvDefiner(Local<Name> property,
+                       const PropertyDescriptor& desc,
+                       const PropertyCallbackInfo<Value>& info) {
+  Environment* env = Environment::GetCurrent(info);
+  if (desc.has_get() || desc.has_set() || desc.has_configurable() ||
+      desc.has_enumerable() || desc.has_writable()) {
+    THROW_ERR_INVALID_DEFINE(env, "Only attribute 'value' is allowed");
```

Comment:
The current wording does not seem to be very infromative. An alternative:

```suggestion
    THROW_ERR_INVALID_DEFINE(env, "Attributes other than 'value' cannot be "
                                  "defined in 'process.env'");
```